src/utils.py

```diff
@@ -98,7 +105,7 @@ def filter_blocks(
 
 def filter_on_block_type(
     input: ParserOutput, remove_block_types: list[str]
-) -> Sequence[ParserOutput]:
+) -> ParserOutput:
```

Comment:
was thas bad typing previously? The outputs doesn't seem to have changed.

Reply:
yes - the typing was wrong given the outputs